pipelines/RLI_pipeline/IUCN_RLI_pipeline.json

```diff
@@ -1151,7 +1151,7 @@
   },
   "metadata": {
     "name": "Red List Index (RLI) pipeline",
-    "description": "This pipeline calculates the \nRLI by filtering the IUCN data by country and taxonomic group.",
+    "description": "## Introduction\n\nThe Red List Index (RLI) shows trends in overall extinction risk for species, \nand is used to track progress towards reducing extinctions and biodiversity loss. \nRLI has been widely integrated into various policy frameworks. Initially used to\nassess progress towards the Convention on Biological Diversity’s 2010 target\n(Rodrigues, 2006), it has since been employed in regional, thematic, and global\nassessments by bodies such as the Intergovernmental Science-Policy Platform on \nBiodiversity and Ecosystem Services, the Global Environment Outlook, and others \n(Global Biodiversity Outlook, 2010).\n\n## Uses\n\nThe RLI is a key indicator for the UN Sustainable Development Goals, particularly\nGoal 15, and is adopted by the Convention on Migratory Species and its agreements.\nIt also serves as a headline indicator for Goal A and Target 4 of the CBD’s Kunming-Montreal \nGlobal Biodiversity Framework (CBD, 2022). Beyond global trends, the RLI can be used to \ntrack trends in species across different biogeographic realms, political units, ecosystems, \nhabitats, taxonomic groups, and those relevant to various international agreements and \ntreaties (Butchart et al., 2004; Butchart et al., 2005).\n\n## Pipeline Limitations\n\n* On larger datasets, the pipeline takes a significant amount of time to load the data.\n\n## Before you start\n\nTo use this pipeline, you’ll need an [IUCN token](https://api.iucnredlist.org/users/sign_up)\nto access data on the International Union for Conservation of Nature (IUCN) Red List of \nThreatened Species.",
```

Comment:
> To use this pipeline, add to the runner.env file your [IUCN token]...

Or the like, to mention runner.env file. I expect people to be lost otherwise!